dataverse-webapp/src/main/webapp/dashboard-users-pager.xhtml

```diff
@@ -11,70 +11,86 @@
         <!-- PAGINATOR --> 
         <div class="col-xs-4">
             <!-- number of results -->
-            <p style="padding:7px 0;margin:0;">
+            <p style="padding:7px 0; margin: 30px 0 0;">
             <h:outputFormat styleClass="highlightBold" value="#{bundle['dataverse.results.count.toofresults']}">
                 <f:param value="#{listPager.startCardNumber}"/>
                 <f:param value="#{listPager.endCardNumber}"/>
                 <f:param value="#{listPager.numResults}"/>
             </h:outputFormat>
             </p>
         </div>
-        <div class="col-xs-8 text-right" jsf:rendered="#{listPager.pagerNecessary}">
-            <ul class="pagination" style="margin:0;">
-                <!-- previous page controls -->
-                <ui:fragment rendered="#{listPager.hasPreviousPageNumber()}">
-                    <li><p:commandLink action="#{DashboardUsersPage.runUserSearchWithPage(1)}" process="@this @widgetVar(inputSearchTerm)" update="@form" partialSubmit="true">                       
+        <div class="text-right col-xs-12">
+            <div class="col-xs-4" style="position: relative">
+                <h:outputFormat style="float: left; margin-top: 8px; margin-left: 150px;" value="#{bundle['dashboard.list_users.usersPerPage.title']}" />
```

Comment:
We shouldn't hardcode px since we have different screen sizes. So we should choose 'relative units' like 'em' or % (this is to all styles below also). Also, I think Marcin said that good practice is to define css in our css file and not in xhtml.

Reply:
Ok fair enough, I just wanna make sure everyone agrees with it since there will be another task from it for Marcin. So I just wanna wait with PR accept till the daily meeting.